### DISCUSSION TITLE
question regarding third_party/setup-nfs.yml

### DISCUSSION THREAD

**Discussion by MagnaFrisia90** [HCL-TECH-SOFTWARE/connections-automation#251] [Q&A] [answered]
Hi,
I am new to ansible deployments & have problems understanding when the nfsSetup.sh-script (https://github.com/HCL-TECH-SOFTWARE/connections-automation/tree/main/roles/third_party/nfs-install/templates/nfsSetupScript)  is executed.
I can not find any reference in the playbooks & I was wondering: Is this something additional or is it actually used/referenced by the playbooks ? if yes: where ? 
I could only "reverse-engineer" it this way :  "playbooks/setup-connections-complete.yml" => "third_party/setup-nfs.yml" => "roles/third_party/nfs-install" => "task install_nfs.yml" & here you can only see a "yum nfs-utils"-call. But no reference to the script.

Kind regards,
Jan

**Comment by sabrina-yee** [comment, marked as answer]
Hi Jan, the `nfsSetup.sh` script is used for manual deployment using the online documentation https://opensource.hcltechsw.com/connections-doc/v8-cr1/admin/install/cp_install_services_tasks.html#section_e4p_jrp_tnb.  

By using this ansible repo, the same NFS setup is done by `setup-nfs.yml` where it uses this [template file](https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/main/roles/third_party/nfs-install/templates/nfs.exports.j2) to generate `/etc/exports` in [this step](https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/main/roles/third_party/nfs-install/tasks/configure_master.yml#L174-L177).
